app/src/main/java/com/github/fribourgsdp/radio/FirestoreDatabase.kt

```diff
@@ -125,6 +125,29 @@ class FirestoreDatabase : Database {
         }
     }
 
+    override fun generateUserId() : Task<Long> {

```

Comment:
What is the use of this method knowing that when we connect to firebase we already have a unique id?

Reply:
It is in case someone doesn't want to connect with google. He gets an id just for the session and can still play with his friends. The difference with a user that registered with google is that he won't have his data saved on the cloud.